README.md

```diff
@@ -3,25 +3,64 @@
 An open-source interoperability standard for tools and content
 management systems used in media production.
 
-OpenAssetIO defines a common set of interactions between a host of the
-API (eg: a Digital Content Creation tool or pipeline script) and an
-Asset Management System.
-
 It aims to reduce the integration effort and maintenance overhead of
 modern CGI pipelines, and pioneer new, standardized asset-centric
-workflows in post-production tooling.
+workflows in content creation tooling.
+
+## Problem Statement
+
+In modern production pipelines, data is often managed by an
+authoritative system (Asset Management System, Digital Asset Manager,
+MAM, et. al).
+
+It is common for media creation tools to reference managed data by its
```

Comment:
I wasn't sure on this, as the intro sentence doesn't talk about how the data is referenced, so it felt like this was more an exposition rather than providing an alternative to a previously stated mechanism?

Reply:
Yeah my intention was to link it to the previous sentence to highlight the dichotomy.  However, the latest rewrite links the two more clearly, so I'll resolve this comment.